### PR TITLE
PR for several issues

### PR DESCRIPTION
**Pull request by StephanMa**
#492 : Archivists (community members) are now able to release charters as long as the charter belongs to the archive 
#516 : hotfix http://monasterium.net/mom/charters-to-publish? (click on "Display changes") crashes the app

---
This change is [Reviewable](https://reviewable.io/reviews/icaruseu/mom-ca/517)

### CODE REVIEW

File: my/XRX/src/mom/app/charter/widget/charter.widget.xml [hunk 1]
```diff
@@ -1340,9 +1331,13 @@ cei-recipient {{
           <xrx:div>my-archive</xrx:div> 
         {      
           (: charter view :)
-          
-            
-           i18n:translate-xml(transform:transform($wcharter:page, $wcharter:xsl, $wcharter:params))
+          let $page :=
```
Comment (GVogeler) [inline my/XRX/src/mom/app/charter/widget/charter.widget.xml]: transposition from `<xrx:expression>` to local variable occurs in https://github.com/icaruseu/mom-ca/blob/bd7c21def18c9bd56c9fdd3b56c073a34e313fcd/my/XRX/src/mom/app/charter/widget/my-charter.widget.xml#L222 as well. Shouldn't we use there the same solution as here?

File: my/XRX/src/mom/app/publication/publication.xqm [hunk 2]
```diff
@@ -23,13 +23,63 @@ We expect VdU/VRET to be distributed in the future with a license more lenient t
...
+};
+
+(: input:
+atomid from a charter : xs:string
+switch indicates returning part of the tokens : xs:string
+
+return:
+indicated token (archive/fond/charterid; collection/charterid)
+:)
+declare function publication:retrieveAtomToken($atomid as xs:string, $switch as xs:string) {
```
Comment (GVogeler) [inline my/XRX/src/mom/app/publication/publication.xqm]: This function seems to be something highly useful everywehere so we might want to put it into charter.xqm ?

File: my/XRX/src/mom/app/charter/widget/charter.widget.xml (hunk 1 above)
Comment (StephanMa) [inline my/XRX/src/mom/app/charter/widget/charter.widget.xml]: @maburg please answer until thursday

File: my/XRX/src/mom/app/publication/publication.xqm (hunk 2 above)
Comment (yngwi) [inline my/XRX/src/mom/app/publication/publication.xqm]: The problem seems to be that there is no real overarching module structure so the place in `charter.xqm` is arguably similarly arbitrary then the function positioning here?

File: my/XRX/src/mom/app/charter/widget/charter.widget.xml (hunk 1 above)
Comment (maburg) [inline my/XRX/src/mom/app/charter/widget/charter.widget.xml]: Certainly no problem

File: my/XRX/src/mom/app/publication/publication.xqm (hunk 2 above)
Comment (GVogeler) [inline my/XRX/src/mom/app/publication/publication.xqm]: 1. namespace `charter:retreiveAtomToken` describes more what the function does than `publication:retrieveAtomToken` - and keeping the namespace synchronized with the app/module-name can help as well.
2. There is the inheritance from `core` to `mom` so we might find a place in `core`?
3. If there is no existing overarching module structure we should think about how to create one at least on the conceptual/intellectual level - and work towards realising it (OK, refactoring is quite a task, I admit).

**Pull request merged**